### ISSUE TITLE
New version is completely broken (Samsung Galaxy S7, Android 7.0)

### ISSUE THREAD

**Issue by AdKiller** [fython/Blackbulb#46]
The new update 2.1.0 doesn't dim the screen, and the quick settings menu button does not appear at all.

(One of the crappy things about Android is that once you update an app you can't easily go back...)

**Comment by NutshellySima** [comment]
@AdKiller I tested v2.1.0 on SM-G9300 (Galaxy S7)/Android 7.0 but couldn't reproduce your issue. It just worked perfectly.

**Comment by michealespinola** [comment]
Agreed. Quick setting tile is missing, and scheduled dim does not seem to work consistently on my Galaxy S8.

**Comment by fython** [comment]
@AdKiller Thanks for your feedback. Sometimes I meet the same problem about QS tiles, I will check it.

If you want to get previous version app, you can uninstall current version (Android is not allowed to downgrade app) and download 1.x from:  [https://github.com/fython/Blackbulb/releases](https://github.com/fython/Blackbulb/releases)

_從我的 ONEPLUS A3010，使用 [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github) 發送。_

**Comment by michealespinola** [comment]
Thank you for looking into it!  I find it extremely useful in low-light environments and at night.  It amazes me that something like this isn't built-in.  The schedule time is nice, but the QS tile integration is super convenient.  If there is anything I can provide/test on my end, I would be happy to help.

**Comment by michealespinola** [comment]
I've since reinstalled the beta, and am no longer having any issues with it.  I did have to manually re-enable the QS tile, but everything is working fine now.
